daras_ai_v2/bot_integration_widgets.py

```diff
@@ -327,33 +326,6 @@ def broadcast_input(bi: BotIntegration):
         gui.button("✅ Yes, Send", key=confirmed_send_btn)
 
 
-def get_bot_test_link(bi: BotIntegration) -> str | None:
-    if bi.wa_phone_number:
-        return (furl("https://wa.me/") / bi.wa_phone_number.as_e164).tostr()
-    elif bi.slack_team_id:
-        return (
-            furl("https://app.slack.com/client")
-            / bi.slack_team_id
-            / bi.slack_channel_id
-        ).tostr()
-    elif bi.ig_username:
-        return (furl("http://instagram.com/") / bi.ig_username).tostr()
-    elif bi.fb_page_name:
-        return (furl("https://www.facebook.com/") / bi.fb_page_id).tostr()
-    elif bi.platform == Platform.WEB:
-        return get_app_route_url(
-            chat_route,
-            path_params=dict(
-                integration_id=bi.api_integration_id(),
-                integration_name=slugify(bi.name) or "untitled",
-            ),
-        )
-    elif bi.twilio_phone_number:
-        return str(furl("tel:") / bi.twilio_phone_number.as_e164)
-    else:
-        return None
-
```

Comment:
refactored to a method on `BotIntegration` because of its usefulness. and other related methods such as `get_display_name()` are there.